tests/build.gradle

```diff
@@ -73,10 +73,11 @@ ext.testSets = [
             "org/apache/openwhisk/core/cli/test/**",
             "org/apache/openwhisk/core/limits/**",
             "org/apache/openwhisk/core/scheduler/queue/test/ElasticSearchDurationCheck*",
+            "org/apache/openwhisk/common/etcd/**",
```

Comment:
Excluded etcd from basic unit tests.

Reply:
So we would add tests for the scheduler when all contribution is over?

Reply:
Yes. One concern is that if the test is excluded, the code is merged without going through the CI test, so I would like to see if the CI also needs a pipeline for the scheduler.